### DISCUSSION TITLE
How create New entry in a Database

### DISCUSSION THREAD

**Discussion by janduplessis883** [ramnes/notion-sdk-py#161] [Q&A] [answered]
How do I create a new entry within a database with **notion-sdk-py**.<BR>
I want to use Notion with Deepnote - saving data to a Notion database.<BR>
Thank you
Jan

**Comment by chapodrou** [comment, marked as answer]
`client.pages.create(**{
        "parent": {
            "database_id": YOUR_DATABASE_ID
        },
        "properties": {
            "title": {
                "title": [
                    {
                        "type": "text",
                        "text": {
                            "content": "name_of_your_entry"
                        }
                    }
                ]
            }
        }
    })`
